color_wallpaper/Color.py

```diff
@@ -184,3 +184,11 @@ def from_str(arg: str) -> "Color":
             rgb = parse_hex(color_to_hex[name])
 
         return Color(rgb)
+
+    @staticmethod
+    def all_colors() -> List["Color"]:
+        """Return a list of Color objects for each color name in data"""
+        colors_list = []
+        for rgb, name in hex_to_color.items():
+            colors_list.append(Color(parse_hex(rgb), name))
+        return colors_list
```

Comment:
Can be shortened to (and it should be faster):

```suggestion
        return [Color(parse_hex(rgb), name) for rgb, name in hex_to_color.items()]
```